### DISCUSSION TITLE
Need help!!

### DISCUSSION THREAD

**Discussion by ItzNovy** [GodotNuts/GodotFirebase#317] [General]
Hello everyone👋🏼I am new to Godot and new to making games in general. I just came here to ask if I could get any help/tell me how to make a random chance game. I am currently working on a side scroll game And would like to implement a casino were players can gamble their coins, but I have no idea how to write the code for a random chance to land on a number (example-kind of looking for something like roulett(

Any help would be greatly appreciated. Thank you. 🖤

**Comment by BearDooks** [comment]
@ItzNovy you should check out this article

https://docs.godotengine.org/en/stable/classes/class_randomnumbergenerator.html

This can help you in the generation of random numbers. From there you can use it to make odds or even pick random cards from an array
